DAO.sol

```diff
@@ -25,68 +26,76 @@ import "./ManagedAccount.sol";
 
 contract DAOInterface {
 
-    // proposals to spend ether of the DAO or choose a new service provider
+    // Proposals to spend the DAO's ether or to choose a new service provider
     Proposal[] public proposals;
-    // the minimal quorum needed for a proposal to vote is calculated by totalSupply / minQuorumDivisor
+    // The quorum needed for each proposal is partially calculated by 
+    // totalSupply / minQuorumDivisor
     uint minQuorumDivisor;
-    // the time of the last proposal which has met the minimal quorum
+    // The unix time of the last time quorum was reached on a proposal
     uint lastTimeMinQuorumMet;
-
-    // to total amount of wei received as reward which has not been sent to the rewardAccount
+    // The total amount of wei received as reward that has not been sent to 
+    // the rewardAccount
     uint public rewards;
-
-    // address of the service provider
+    // Address of the service provider
     address public serviceProvider;
-    // list of addresses the DAO is allowed to send money to
+    // The whitelist: List of addresses the DAO is allowed to send money to
     address[] public allowedRecipients;
 
-    // only used for splits, give DAOs without a balance the privilege to access their share of the rewards
-    // conceptually, rewardTokens represent a share of right to receive rewards that arise from the already spent fund.
+    // Tracks the addresses that own Reward Tokens. Those addresses can only be 
+    // DAOs that have split from the original DAO. Conceptually, Reward Tokens 
+    // represent the proportion of the rewards that the DAO has the right to 
+    // receive. These Reward Tokens are generated when the DAO spends ether.
     mapping (address => uint) public rewardToken;
-    // sum of all rewardToken
+    // Total supply of rewardToken
     uint public totalRewardToken;
 
-    // account used to manage the rewards which are to be distributed to the
-    // DAO Token Holders (and reward token holders) separately, so they don't appear in `this.balance`
+    // The account used to manage the rewards which are to be distributed to the
+    // DAO Token Holders of any DAO that holds Reward Tokens 
     ManagedAccount public rewardAccount;
-    // amount of wei already paid out to a certain member address
+    // Amount of rewards (in wei) already paid out to a certain address
     mapping (address => uint) public paidOut;
-    // map of address blocked during a vote (not allowed to transfer tokens). The address points to the proposal ID.
+    // Map of addresses blocked during a vote (not allowed to transfer DAO  
+    // tokens). The address points to the proposal ID.
     mapping (address => uint) public blocked;
 
-    // deposit in wei to be held for each proposal
+    // Deposit in wei the creator added when submitting their proposal
```

Comment:
since there can be many proposals at the same time, this is wrong.
Use this: "Minimal deposit for a regular proposal" or something similar.